examples/apps/ping/src/icmpv4.adb

```diff
@@ -210,7 +210,9 @@ is
          RFLX.ICMP.Message.Set_Data (ICMP_Context, Data);
          Last := RFLX.RFLX_Types.To_Index (RFLX.ICMP.Message.Message_Last (ICMP_Context));
          RFLX.ICMP.Message.Take_Buffer (ICMP_Context, Buf);
-         Sequence := Sequence + 1;
+         Sequence := (if Sequence < RFLX.ICMP.Sequence_Number'Last
+                      then Sequence + 1
+                      else RFLX.ICMP.Sequence_Number'First);
```

Comment:
That's a bit concerning that this integer overflow slipped though. @treiher Maybe we should capture `gnatprove`s output and grep for errors/warnings?

Reply:
It shouldn't be necessary with newer SPARK versions, but we could do that to be on the safe side. Do you see a simple method to do that? It would be nice, if the output would still be visible at runtime and not just after gnatprove finishes. `gnatprove ... | tee /dev/tty | grep -v "warning\|error\|low\|medium\|high"` works for me locally, but does not work in the GitHub CI, as `/dev/tty` cannot be found (i.e., the command is still executed, but there will be no output in the CI log).

Reply:
[This answer](https://unix.stackexchange.com/a/625189) seems to do what we're interested in using standard tools (not that I understand what's happening there... :stuck_out_tongue_winking_eye:).

Reply:
Nice find! The additional safety net is in place now. I was also able to replace the existing problematic uses of `... | tee /dev/tty | grep ...`.